GVFS/FastFetch/FastFetchVerb.cs

```diff
@@ -135,6 +135,13 @@ public class FastFetchVerb
             HelpText = "The GUID of the caller - used for telemetry purposes.")]
         public string ParentActivityId { get; set; }
 
+        [Option(
+            "force",
+            Required = false,
+            Default = false,
+            HelpText = "Force FastFetch to download content as if the current repository was just initialized.")]

```

Comment:
It's not really "downloading" that's the issue, it's checking out the files.This flag only does anything if you also specify `--checkout` right? I think that also suggests that the flag should be called `--force-checkout`

Reply:
:+1:

Reply:
Hmm, currently it controls the diff for both the `--checkout` and non checkout cases. There are 2 blob fetchers for each case. This usage is really just for checkout though so I like the idea of calling it `--force-checkout` and then limiting it to the checkout path.